### ISSUE TITLE
Add quicklook guide to hub private? 

### ISSUE THREAD

**Issue by jackiekazil** [18F/training#34]
After the quicklook guide is confirmed, figure out where to add quick look guide to hub. 
(Current location of quicklook guide -- https://docs.google.com/a/gsa.gov/document/d/1etenyqx3tZrN8Jmxhlmpzaai0CsCLfJ-4mRfnnKG0LQ/edit)

**Comment by jackiekazil** [comment]
@PiaScott is this read to be added to Hub?

**Comment by jackiekazil** [comment]
@jenniferthibault ... the virtual one isn't redundant... but the in person one. I just saw the pull request & @mbland's message. @PiaScott created the quick look guide. We just proposed the path forward today.

I **think** we are ready for publishing in person on hub, but we were holding off, because there were a lot of nuances to it. I am going to defer to @PiaScott  & @StephanieRivera to make the call on whether we should publish this. 

We have had issues in the past jumping the gun on training... so if we are not ready, we might want to take that page down in Hub.
